src/bullscows/SecretCodeGenerator.java

```diff
@@ -0,0 +1,98 @@
+package bullscows;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class SecretCodeGenerator {
+
+    int inputLengthOfCode() {
```

Comment:
Can we move all input methods (inputLengthOfCode, inputNumberOfSymbols) to InputScanner?